wasm/src/lib.rs

```diff
@@ -28,6 +28,31 @@ pub fn git_hash() -> JsValue {
     JsValue::from_str(git_hash)
 }
 
+#[wasm_bindgen]
+pub fn provide_text_document_content(name: &str) -> JsValue {
+    thread_local! {
+        static STORE_CORE_STD: (PackageStore, PackageId) = {
+            let mut store = PackageStore::new(compile::core());
+            let std = store.insert(compile::std(&store));
+            (store, std)
+        };
+    }
```

Comment:
Since this is the same thing that the `compile` function does below, can we just pull this out into the global scope in this file so that the two functions can share the instance?